### PR TITLE
Original content - adds a new toy inspired by goonstation

### PR DESCRIPTION
**Pull request by jbox144**
## About The Pull Request

Have you ever wanted to be a coder, but don't want to navigate the maze of spaghetti code that SS13 is built on? Never fear! Introducing the aucode hard disk drive! Simply swipe your PDA on it and upload whatever PR content you want*

*Does not upload content. Batteries not included.

## Why It's Good For The Game

A good meme and a satirical jab to all the stolen content we've amassed here.

## Changelog
:cl:
add: Added the aucode hard disk drive, found in tech storage. Upload whatever you want!
/:cl:

### CODE REVIEW

**Comment by TheFakeElon** [comment]
Obligatory: have you taken **any** assets, code or structure from goonstation in the making of this PR

File: austation/code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -0,0 +1,67 @@
+/obj/item/toy/aucode // CTRL + C, CTRL + V of the "gooncode" toy from, well, goon. Heavily modified for beecode, naturally.
```
Comment (TheFakeElon) [inline austation/code/game/objects/items/toys.dm:1]: :(

**Comment by TheFakeElon** [comment]
Please read our contributor guidelines. Ripping anything from goonstation, especially code, is an AGPL violation (could get our repo nuked).

You can't call it "inspired" if you copypaste code. Unauthorized goon ports are grounds for a repository ban but by the looks of it you didn't know about it so I'll let it slide. Please read contributing.md before making any further contributions.

**Comment by Singularity69** [comment]
:(

**Pull request closed without merging**